### PR TITLE
Refactor ingredient info

### PR DESCRIPTION
**Pull request by hermancai**
This branch refactors the 'getIngredientInfo' route into multiple routes (getEthicalProblem, getEthicalDescription, getAlternatives).
It also includes the feature to view ethical problems per each ingredient in a recipe.

### CODE REVIEW

**Review by anthonyliuhao — DISMISSED** [review]

**Review by alexlo510 — DISMISSED** [review]
nice work Herman

File: main.js [hunk 1]
```diff
@@ -164,7 +164,8 @@ app.get('/recipe', function (req, res, next) {
               inner join ingredient i on i.id = ip.ingredient_id
               inner join ethical_problem p on p.id = ip.problem_id
               inner join ethical_description ee on ee.id = ip.explain_id
-              where i.id = any ($1)`,
+              where i.id = any ($1)
+              order by i.name asc`,
```
Comment (anthonyliuhao) [inline main.js]: It is a little confusing that the ethical issue column is sorted by ingredient name but the ingredient list itself is not sorted.

**Review by anthonyliuhao — CHANGES_REQUESTED** [review]
I just realized how we had completely different ideas in mind when we talked about making the ethical issues more obvious.

Would it be possible to implement it so that there is a warning icon next to every ingredient with an ethical issue? When the user hovers over the icon, a tooltip is displayed with a description of the ethical issue.
Blank diagram

The current design is a little overwhelming. An Abi or Pat will feel obligated to read everything before they do anything. This goes against the following.
> Heuristic #3 (of 9): Let people gather as much information as they want, and no more than they want

**Review by anjelbenitez — APPROVED** [review]
Looks good!

**Review by anthonyliuhao — APPROVED** [review]
Great work. Thanks @hermancai !

**Pull request merged**